app/views/static_pages/contributing.html.erb

```diff
@@ -49,32 +49,22 @@
         </div>
 
         <br/>
-        <div class="row">
-          <div class="col-md-8 offset-md-2 mb-5">
-            <p class="text-muted">
-              Instructions on setting up your local development environment can be found in the <%= link_to 'GitHub repo', github_link, target: '__blank' %>.
-            </p>
-            <p class="text-muted">
-              <em>NOTE: When choosing an issue to work on, please check out the <%= link_to '#contributing', 'https://gitter.im/TheOdinProject/Contributing', target: '_blank' %> channel on our Gitter chat, and let everyone know what you are working on.  This is important, as somebody else may have already done some work on the issue - in which case they would probably love to have your help!</em>
-            </p>
-          </div>
-        </div>
+      
       </section>
     </div>
   </div>
+    <%
```

Comment:
@arun1595 I think its better to have this on the actual view imo. Having it in the application helper means you have to go to another file to find it and we shouldn't really be filling the application helper with these kind of methods.